README.md

```diff
@@ -279,38 +279,10 @@ The unit_test makefile target is used for running the unit tests:
 
 Please have a look to the section [on building the source code](#from-sources) in order to get more information about how to prepare the environment to run the unit_test target.
 
-## Advanced topics:
-
-* Installation and administration
-  * [Building from sources](doc/manuals/admin/build_source.md)
-  * [Running Orion from command line](doc/manuals/admin/cli.md)
-  * [Database administration](doc/manuals/admin/database_admin.md)
-  * [Logs](doc/manuals/admin/logs.md)
-  * [Watchdog](doc/manuals/admin/watchdog.md)
-  * [Rush relayer](doc/manuals/admin/rush.md)
-  * [Management REST inferface](doc/manuals/admin/management_api.md)
-  * [Data model](doc/manuals/admin/database_model.md)
-  * [Resources & I/O Flows](doc/manuals/admin/resources.md)
-  * [Problem diagnosis procedures](doc/manuals/admin/diagnosis.md)
-* API
-  * [Pagination](doc/manuals/user/pagination.md)
-  * [Geolocation ](doc/manuals/user/geolocation.md)
-  * [Structured values for attributes](doc/manuals/user/structured_attribute_valued.md)
-  * [Context Providers registration and request forwarding](doc/manuals/user/context_providers.md)
-  * [Attribute metadata](doc/manuals/user/metadata.md)
-  * [Filtering results](doc/manuals/user/filtering.md)
-  * [Multy tenancy](doc/manuals/user/multitenancy.md)
-  * [Entity service paths](doc/manuals/user/service_path.md)
-  * [Adding and removing attributes and entities with APPEND and DELETE in updateContext](doc/manuals/user/append_and_delete.md)
-  * [Updating registrations and subscriptions](doc/manuals/user/updating_regs_and_subs.md)
-  * [Context broker federation](doc/manuals/user/federation.md)
-  * [Forbidden characters](doc/manuals/user/forbidden_characters.md)
-  * [Duration (for registration and subscriptions)](doc/manuals/user/duration.md)
-  * [Using empty types](doc/manuals/user/empty_types.md)
-  * [Mixing standard and convenience operations](doc/manuals/user/std_conv_mix.md)
-  * [HTTP and NGSI response codes](doc/manuals/user/http_and_ngsi_sc.md)
-  * [Security considerations](doc/manuals/user/security.md)
-  * [Known limitations](doc/manuals/user/known_limitations.md)
+## Advanced topics
+
+* [Installation and administration](http://fiware-orion.readthedocs.org/en/develop/admin/index.html)
```

Comment:
Mixing links to readthedocs documentation and Markdown documentation could be a bit confusing from the point of view of the user of the README.md... I'd suggest:

1. Add the link to readthedocs to Documentation tab in the catalogue (http://catalogue.fiware.org/enablers/publishsubscribe-context-broker-orion-context-broker/documentation)
2. Add the link to readthedocs at the top of the README.md file (in the same way we add there the link to the Orion page in the FIWARE catalaogue, the link to the FIWARE web site and the links to the old manuals in the wiki) introducing it in the following way (or something similar): "A version of this documentation rendered in the readthedocs format can be found at \<link\>".